airbnb_project/listings/listing_models.py

```diff
@@ -1,16 +1,19 @@
 from django.db import models
 
 class Listing(models.Model):
-  name = models.CharField(max_length=100)
-  baths = models.CharField(max_length=10)
-  beds = models.CharField(max_length=10)
-  latitude = models.CharField(max_length=20)
-  location = models.CharField(max_length=20)
-  longitude = models.CharField(max_length=20)
-  person_capacity = models.CharField(max_length=20)
-  registration_number = models.CharField(max_length=20)
-  room_type = models.CharField(max_length=20)
-  title = models.CharField(max_length=100)
+    airbnb_listing_id = models.TextField(blank=False, null=False)
+    name = models.TextField(null=True, blank=True)
+    baths = models.TextField(null=True, blank=True)
+    beds = models.TextField(null=True, blank=True)
```

Comment:
Should these be text fields?

Reply:
Same reason as before, just want to be safe and store what ever is returned by airbnb.

Reply:
I think that we need to create the dimensions/features that'd be most relevant to the downstream systems. We can chat more in the team meeting about the role of harvester in the entire pipeline so that we are all on the same page.

Reply:
yeah, we can discuss more regarding this. Previously, we did have a brief discussion that harvester shouldn't really modify data and that downstream modules should do their own validation etc.